### DISCUSSION TITLE
Práctica HTML Form

### DISCUSSION THREAD

**Discussion by LJimena70** [Launch-X-Latam/MisionFrontEnd#331] [Q&A] [answered]
Hola, buena tarde:
Se mencionó que hoy se publica el Form para anexar la liga de la práctica pero no logro encontrar el form...
¿Alguien tiene el enlace? o ¿Solamente se sube a nuestro repositorio?

Gracias de antemano :)
![Captura](https://user-images.githubusercontent.com/98842390/200089029-e70f47d5-1a96-4d7b-a241-d7a475c200ba.JPG)

**Comment by MarioEduardoAlvarezTorres** [comment]
Ando esperando lo mismo

**Comment by iam-leo** [comment, marked as answer]
No hay form. Todas las prácticas se suben a sus github personales!

**Reply by LJimena70** [reply]
Gracias! :-)

**Reply by iam-leo** [reply]
De nada :slightly_smiling_face:
